src/stylesString.js

```diff
@@ -1,37 +1,30 @@
 'use strict';
 
 const stylesString = `
-  background-color:      #fff;
--webkit-border-radius: 5px;
-  border-radius     : 5px;
+  background-color: #fff;
+  -webkit-border-radius: 5px;
+  border-radius : 5px;
   border: 1px solid #e8e8e8;
   -webkit-box-sizing: border-box;
   box-sizing: border-box;
-  clear   : both       ;
```

Comment:
Don't change this file. You have to process all these values with code

Reply:
rollback this changes